content/docs/command-reference/params/index.md

```diff
@@ -47,7 +47,7 @@ dependency (e.g. a config file), and any change in it invalidates all of them
 (see `dvc status`), causing unnecessary re-executions upon `dvc repro`.
 
 The default **parameters file** name is `params.yaml`, but any other YAML 1.2,
-JSON, TOML, or [Python](#examples-python-parameters-file) files can be used
+JSON, TOML 1.0, or [Python](#examples-python-parameters-file) files can be used
 additionally (listed under `params:` as shown in the sample above). These files
```

Comment:
Thanks! There's another instance we missed in https://dvc.org/doc/command-reference/metrics#supported-file-formats

Reply:
https://github.com/iterative/dvc.org/pull/3871